### ISSUE TITLE
paddle.layer.warp_ctc 加权重训练出错

### ISSUE THREAD

**Issue by gaoyiyeah** [PaddlePaddle/Paddle#14772]
由于实验中需要给不同的ctc_loss加权重，但发现会报错
例如：
import paddle.v2 as paddle
ctc_loss = paddle.layer.warp_ctc(input,label,size,blank)         可以正常运行
ctc_loss = 1.0 * paddle.layer.warp_ctc(input,label,size,blank) 运行则报错：
F1206 17:19:14.852391 16845 SlopeInterceptLayer.cpp:69] Check failed: size == inV->getWidth() (29 vs. 1) 
*** Check failure stack trace: ***
    @     0x7f903d92ef2d  google::LogMessage::Fail()
    @     0x7f903d9329dc  google::LogMessage::SendToLog()
    @     0x7f903d92ea53  google::LogMessage::Flush()
    @     0x7f903d933eee  google::LogMessageFatal::~LogMessageFatal()
    @     0x7f903d61a694  paddle::SlopeInterceptLayer::forward()
    @     0x7f903d5cf4ad  paddle::NeuralNetwork::forward()
    @     0x7f903d5acc94  paddle::TrainerThread::forward()
    @     0x7f903d5adef5  paddle::TrainerThread::computeThread()
    @     0x7f9066a8d8f0  (unknown)
    @     0x7f908a3e76db  start_thread
    @     0x7f908a72188f  clone
    @              (nil)  (unknown)
Aborted (core dumped)

**Comment by NHZlX** [comment]
v2 中应该不可以将layer的输出和数字相乘, @reyoung  v2中有像fluid中scale样子的接口么。

**Comment by lucywsq** [comment]
您好，此问题超过48小时未有进一步回复，我们将于今天内关闭，如果您若在关闭后您仍需跟进提问，可重新开启此问题，我们将在24小时内回复您。因关闭带来的不便我们深表歉意，请您谅解~感谢您对PaddlePaddle的支持
